test/src/integration/mempool.test.ts

```diff
@@ -331,18 +331,17 @@ describe("Timelock", () => {
             recipient: await node.createP2PKHAddress()
         });
         await node.signTransferInput(tx, 0);
-        await node.sendTransaction(tx, { awaitInvoice: false });
+        try {
+            await node.sendTransaction(tx, { awaitInvoice: false });
+            done.fail();
+        } catch (e) {
+            expect(e.data).toContain("Timelocked");
```

Comment:
I changed the expected result that depends on the failed transaction.

Reply:
Ok.